core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/DataRow.kt

```diff
@@ -28,13 +28,16 @@ public interface DataRow<out T> {
 
     public operator fun <R> get(expression: RowExpression<T, R>): R = expression(this, this)
 
+    @AccessApiOverload
     public operator fun <R> get(column: ColumnReference<R>): R
```

Comment:
hmm maybe not exclude these ones yet? Let's say we have a `DataRow<Any>`, calling `row["name"]` provides no way of specifying the type. You can however do `row[column<String>("name")]` to get the type. (potentially even `row["name"<String>()]`) It would also suffice if we could get a typed `row.get<String>("name")` function I believe.

Reply:
we have `typed.first().getValue<String>("name")`